tracks/haskell/exercises/leap/mentoring.md

```diff
@@ -31,7 +64,9 @@ isLeapYear year
   | otherwise           = True
 ```
 
-This solution evaluates the three conditions in an optimal order.
+This solution evaluates the three conditions in an optimal order at the cost of readability.
+
+Since guards may be introduced to improve readability, this seems a little counterproductive.
```

Comment:
What does `this` refer to? This can refer to this readability, or implicitly "this introduction of guards", whereas you might mean "this solution". If so, repeat the word solution.

```suggestion
Since guards may be introduced to improve readability, this solution seems a little counterproductive.
```